gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -164,13 +164,14 @@ class BaselineIdea extends AbstractBaselinePlugin {
         def runManager = node.component.find { it.'@name' == 'RunManager' }
 
         if (runManager) {
-            // if RunManager configuration exists, set all WORKING_DIRECTORY options to module
-            def items = runManager.configuration.option.findAll { it.'@name' == 'WORKING_DIRECTORY' }
-            items.each { item ->
-                item.'@value' = 'file://$MODULE_DIR$'
-            }
+            // if RunManager configuration exists, set Application, JUnit WORKING_DIRECTORY default to $MODULE_DIR$
+            def appJunitDefaults = new NodeList(runManager.configuration
+                    .findAll { it.'@default' == 'true' && it.'@type' in ['Application', 'JUnit'] })
```

Comment:
can you pull out a variable `def configurations = ['Application', 'JUnit']` and use it here and in the else block?